website/src/models/group.js

```diff
@@ -98,30 +98,30 @@ schema.statics.sanitizeUpdate = function sanitizeUpdate (updateObj) {
 }*/
 
 // TODO test
-schema.pre('remove', true, function preRemoveGroup (next, done) {
+schema.pre('remove', true, async function preRemoveGroup (next, done) {
```

Comment:
Hmm I hadn't thought using async functions here would have worked... The only thing is that it's not inside a try/catch block which means errors will be swallowed. You should wrap the function body in a try/catch block and in case of errors pass the error to `done(err)`